README.md

```diff
@@ -13,6 +13,20 @@ or connect it to the other parts of the Biomage Single Cell Platform.
 
 We hihgly recommend using VSCode for local development. Make sure you also have `npm` and `docker` installed.
 
+You will also need to have aws command line interface `aws-cli` installed and cofigured. Install it with
```

Comment:
I used `brew install awscli`. This is what I would suggest as the installation method, because it is simpler and also makes keeping up to date simpler (`brew upgrade awscli`)

Reply:
I used the recommended method from aws ([link](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-mac.html)). They dont' mention brew so I think it's more probable that it will be outdated. In my opinion it's better to use the recommended install method as it is less probable that it will break or be out of date (and two lines instead of one don't seem too much effort for me). However, I don't have a strong opinion so I'm ok with changing it. I don't know much about homebrew other than it is community mantained.

Reply:
I actually realized we were already recommending to install it below so I changed it to use the same format (just pointing to the link).